### ISSUE TITLE
annot.setRect(rect) as no effect on text Annotation

### ISSUE THREAD

**Issue by PCarle** [pymupdf/PyMuPDF#421]
Text annotation created by page.addTextAnnot() method have a strange behavior on settRec (no zoom and strange drawing on resize) :
<pre> 
<code="Python">
# -*- coding: utf-8 -*-
from __future__ import print_function
import sys
print("Python", sys.version, "on", sys.platform, "\n")
import fitz
print(fitz.__doc__, "\n")

import os
print (os.getcwd())
from matplotlib.pyplot import imshow
import numpy as np
from PIL import Image

def print_descr(rect, annot):
    """Print a short description to the right of an annot rect."""
    annot.parent.insertText(rect.br + (10, 0),
                    "'%s' annotation" % annot.type[1], color = red)

red    = (1, 0, 0)
blue   = (0, 0, 1)
gold   = (1, 1, 0)
colors = {"stroke": blue, "fill": gold}
border = {"width": 1.3}
displ = fitz.Rect(0, 50, 0, 50)
r = fitz.Rect(50, 100, 220, 135)
t1 = u"têxt üsès Lätiñ charß,\nEUR: €, mu: µ, super scripts: ²³!"

doc = fitz.open()
page = doc.newPage()

annot = page.addTextAnnot(r.tl, t1,icon="Tag")

baserect=annot.rect
rect=annot.rect
print ("annot rect",rect)
rect.x1=rect.x0+40
rect.y1=rect.y0+40
annot.setRect(rect)
annot.setOpacity(0.5)
annot.update()
print_descr(annot.rect, annot)
r+=displ
annot = page.addTextAnnot(r.tl, t1,icon="Tag")
print_descr(annot.rect, annot)

shape=page.newShape()
baserect= baserect+ (50, 0, 50, 0)
shape.drawPolyline([baserect.bl, baserect.br, baserect.tr,baserect.tl,baserect.bl])
shape.finish(width=2, color=red, fill=(26./255., 255./255., 255./255.))
rect=rect+(100,0,100,0)
shape.drawPolyline([rect.bl, rect.br, rect.tr,rect.tl,rect.bl])
shape.finish(width=2, color=blue, fill=(255./255., 26./255., 255./255.))
shape.commit(False)

pix=page.getPixmap(matrix=fitz.Matrix(fitz.Identity).preScale(3,3),alpha=True,colorspace="RGBA")

from PIL import Image, ImageTk
mode = "RGBA"
img = Image.frombytes(mode, [pix.width, pix.height], pix.samples)

from matplotlib import pyplot as plt
import matplotlib as mpl
mpl.rc("savefig", dpi=300)
plt.figure(figsize = (20,20))
fig, ax = plt.subplots(figsize=(18, 20))
ax.imshow(np.asarray(img))
</code>
</pre>
![draw pymupdf 1 16 9_3](https://user-images.githubusercontent.com/16350743/71260715-d0b84000-233b-11ea-9bc0-78f4da054056.png)

**Comment by JorjMcKie** [comment]
The reason for this is, that the update method invokes the MuPDF update function ... which re-establishes the standard geometric layout for this (and some other) annot types.
In effect, the rectangle will again have then receive the standard size again (but may be dislocated).

As per current, you can avoid this by executing ``setRect`` **_after_** (or without) executing the ``update`` method.

I will intercept this behavior and prevent MuPDF from doing this ...

Thanks for bringing this up!

**Comment by JorjMcKie** [comment]
fixed with v1.16.10, which should be available on PyPI within minutes

**Comment by PCarle** [comment]
Well each time i update annotation, i need to reset rectangle just after !
```python
rect=annot.rect
annot.setColors({"stroke": blue})
annot.update()
annot.setRect(rect)
```
![reshape TextAnnot](https://user-images.githubusercontent.com/16350743/77169593-05ae6d00-6aba-11ea-9323-e6c1658141f5.jpg)


Just a remark in 1.16.13 the annotation don't accept opacity (uncomment annot.setOpacity(0.5))
```
![reshape TextAnnot](https://user-images.githubusercontent.com/16350743/77169555-f8917e00-6ab9-11ea-919a-9234d34de741.jpg)
python
<code="Python">
from __future__ import print_function
import sys
print("Python", sys.version, "on", sys.platform, "\n")
import fitz
print(fitz.__doc__, "\n")

import os
print (os.getcwd())
from matplotlib.pyplot import imshow
import numpy as np
from PIL import Image


def print_descr(rect, annot):
    """Print a short description to the right of an annot rect."""
    annot.parent.insertText(rect.br + (10, 0),
                    "'%s' annotation" % annot.type[1], color = red)


red    = (1, 0, 0)
blue   = (0, 0, 1)
gold   = (1, 1, 0)
colors = {"stroke": blue, "fill": gold}
border = {"width": 1.3}
displ = fitz.Rect(0, 50, 0, 50)
r = fitz.Rect(50, 100, 220, 135)
t1 = u"têxt üsès Lätiñ charß,\nEUR: €, mu: µ, super scripts: ²³!"

doc = fitz.open()
page = doc.newPage()

annot = page.addTextAnnot(r.tl, t1,icon="Tag")

baserect=annot.rect
rect=annot.rect
print ("annot rect",rect)
rect.x1=rect.x0+40
rect.y1=rect.y0+40
annot.setRect(rect)
#annot.setOpacity(0.5)
annot.update()
print_descr(annot.rect, annot)
r+=displ
annot = page.addTextAnnot(r.tl, t1,icon="Comment")
print_descr(annot.rect, annot)

shape=page.newShape()
baserect= baserect+ (50, 0, 50, 0)
shape.drawPolyline([baserect.bl, baserect.br, baserect.tr,baserect.tl,baserect.bl])
shape.finish(width=2, color=red, fill=(26./255., 255./255., 255./255.))
rect=rect+(100,0,100,0)
shape.drawPolyline([rect.bl, rect.br, rect.tr,rect.tl,rect.bl])
shape.finish(width=2, color=blue, fill=(255./255., 26./255., 255./255.))
shape.commit(False)

pix=page.getPixmap(matrix=fitz.Matrix(fitz.Identity).preScale(3,3),alpha=True,colorspace="RGBA")

from PIL import Image, ImageTk
mode = "RGBA"
img = Image.frombytes(mode, [pix.width, pix.height], pix.samples)

from matplotlib import pyplot as plt
import matplotlib as mpl
mpl.rc("savefig", dpi=300)
plt.figure(figsize = (20,20))
fig, ax = plt.subplots(figsize=(18, 20))
ax.imshow(np.asarray(img))
```
